src/Core__Float.resi

```diff
@@ -1,26 +1,404 @@
+See [`Number.toPrecision`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) on MDN.
+
+## Examples
+
+```rescript
+Float.toPrecision(100.0) // "100"
+Float.toPrecision(1.0) // "1"
+```
+*/
+@send
+external toPrecision: float => string = "toPrecision"
+
+/**
+`toPrecision(v, ~digits)` return a `string` representing the giver value with
+precision. `digits` specifies the number of significant digits.
+See [`Number.toPrecision`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) on MDN.
+
+## Examples
+
+```rescript
+Float.toPrecision(100.0, ~digits=2) // "1.0e+2"
+Float.toPrecision(1.0) // "1.0"
+```
+
+## Exceptions
+
+- `RangeError`: If `digits` is not between 1 and 100 (inclusive).
+Implementations are allowed to support larger and smaller values as well.
+ECMA-262 only requires a precision of up to 21 significant digits.
+
+*/
+@send
+external toPrecisionWithPrecision: (float, ~digits: int) => string = "toPrecision"
+
+/**
+`toString(v)` return a `string` representing the given value.
+See [`Number.toString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString) on MDN.
+
+## Examples
+
+```rescript
+Float.toString(1000.0) // "1000"
+Float.toString(-1000.0) // "-1000"
+```
+*/
+@send
+external toString: float => string = "toString"
+
+/**
+`toStringWithRadix(v, ~radix)` return a `string` representing the given value.
+`~radix` specifies the radix base to use for the formatted number.
+See [`Number.toString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString) on MDN.
+
+## Examples
+
+```rescript
+Float.toString(6.0, ~radix=2) // "110"
+Float.toString(3735928559.0, ~radix=16) // "deadbeef"
+Float.toStringWithRadix(123456.0, ~radix=36) // "2n9c"
+```
+
+## Exceptions
+
+`RangeError`: if `radix` is less than 2 or greater than 36.
+*/
+@send
+external toStringWithRadix: (float, ~radix: int) => string = "toString"
+
+/**
+`toLocaleString(v)` return a `string` with language-sensitive representing the
+given value.
+See [`Number.toLocaleString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) on MDN.
+
+## Examples
+
+```rescript
+// If the application uses English as the default language
+Int.toLocaleString(1000.0) // "1,000"
+
+// If the application uses Portuguese Brazil as the default language
+Int.toLocaleString(1000.0) // "1.000"
+```
+*/
+@send
+external toLocaleString: float => string = "toLocaleString"
+
+/**
+`fromString(~radix?, str)` return an `option<int>` representing the given value
+`str`.
+
+## Examples
+
+```rescript
+Float.fromString("0") == Some(0.0)
+Float.fromString("NaN") == None
+Float.fromString("6") == None
+```
+*/
 let fromString: 'a => option<float>
+
+/**
+`toInt(v)` returns an int to given float `v`.
+
+## Examples
+
+```rescript
+Float.toInt(2.0) == 2
```

Comment:
Might be interesting to have an example for what happens when you pass something with a fraction, like 1.1 or 1.6.

Reply:
Fixed c49fcdf